apps/backend/prisma/schema.prisma

```diff
@@ -68,6 +69,7 @@ model Message {
   senderEncryptedSignaturePublicKey  String
   senderEncryptedSignature           String
   createdAt                          DateTime @default(now())
+  type                               String? // Null type evaluates to TAP_BACK type
```

Comment:
We should not add type to the `Message` table. See tap backs - the message type is included in the blob which is encrypted and sent to the other user. This ensure privacy at the message level

Reply:
I was confused by that -- how do you know the type of the message so that you can properly parse it?

Reply:
you would decrypt the message, then you parse out the type field, and then based on that you parse it accordingly. this is not yet implemented, should be added here: https://github.com/cursive-team/connections/blob/a6748732dc8053ef564ef0205ea3e17e04985f7a/apps/frontend/src/lib/storage/localStorage/user/connection/message/index.ts#L49

Reply:
Addressed.